open_academy/wizard/open_academy_wizard.py

```diff
@@ -0,0 +1,16 @@
+from odoo import models, fields
+
+
+class OpenAcademyWizard(models.TransientModel):
+    _name = 'openacademy.wizard'
+    _description = 'Wizard model'
+
+    session_id = fields.Many2one('openacademy.session', default=lambda self: self._default_sessions())
+    attendees_ids = fields.Many2many('res.partner')
+
+    def _default_sessions(self):
```

Comment:
Pending this https://github.com/CDanielSanSeg/OpenAcademyCourse/pull/3#discussion_r995203919